airflow/serialization/serialized_objects.py

```diff
@@ -202,7 +203,11 @@ def _deserialize(cls, encoded_var):  # pylint: disable=too-many-return-statement
             return [cls._deserialize(v) for v in encoded_var]
 
         assert isinstance(encoded_var, dict)
-        var = encoded_var[Encoding.VAR]
+        try:
+            var = encoded_var[Encoding.VAR]
+        except KeyError as e:
+            logging.info("encoded_var: %s", encoded_var)
```

Comment:
What's this line for? (Being at info seems like the wrong level)

Reply:
Removed